our-work/workshops-and-talks/remote_renderer

```diff
@@ -0,0 +1,18 @@
+title: 3D Remote Renderer on Oscar
+description: Get to know this new service to use Oscar as your own 3D render server to visualize datasets that will not fit in your desktop/laptop. We will use Paraview as a tool to interact and analyze your data.
+slug: remote-renderer
+languages:
+  - unix-cli
+tags:
+  - visualization
+  - oscar
+people:
+  - name: Camilo Diaz
+    github_user: kmilo9999
+groups:
+  - dscov
+links:
+  - category: video 
+    url: https://brown.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=526cb8b8-2ce3-439b-ab84-adc001569019
+  - category: documentation 
```

Comment:
there isn't a "documentation" category in the enum – `enum('code', 'slides', 'website', 'video')`

Reply:
How should I categorize the documentation ? Is website ok?  The url points to the remote render in oscar documentation

Reply:
We can just add `documentation` to the enum

Reply:
Actually, John fixed this in his PR, which I just merged

Reply:
@hollandjg @eldu should we leave the tag as `website` or add `documentation` to the enum. I'm not sure which makes more sense

Reply:
I think either would work !